src/front/app/components/Create/Body.js

```diff
@@ -0,0 +1,84 @@
+import React, { PropTypes } from 'react';
+import Category from './category';
+
+const categories = [
+    { name: 'Category 1213'},
+    { name: 'Category 2'},
+    { name: 'Category 3'},
+    { name: 'Category 4'}
+];
+
+const Body = ({title, description}) => (
+    <div className="challenge-create__body">
+        <div className="challenge-create__photo">
+            <button className="challenge-create__photo-download">Download</button>
+            <img className="challenge-create__image" src="" />
+        </div>
+
```

Comment:
I would like to propose use component name for main wrapper of the component, and shorter class names for its blocks. For example main wrapper css class - challenge-create, and inside it blocks with class names: challenge__image / challenge__body etc.